package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "babel-eslint": "4.1.3",
-    "eslint": "codeclimate/eslint.git#9dba1fe",
+    "eslint": "codeclimate/eslint.git#d24d0b451b75cfad304faeb394371096f2c90777",
```

Comment:
No real reason - I brought it up because that's what we've done historically here